src/pycram/process_modules/pr2_process_modules.py

```diff
@@ -80,6 +81,7 @@ def _execute(self, desig: MoveGripperMotion):
             robot.set_joint_position(joint, state)
 
 
+@DeprecationWarning
 class Pr2Detecting(ProcessModule):
```

Comment:
You can delete both Detecting process modules since they are not used anymore but good idea with the DeprecationWarning